how-to/how-to-template-guide.md

```diff
@@ -0,0 +1,149 @@
+
+| Tutorials                                                                                                                                       | How-to                                                                                                   |
+| ----------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| Learning oriented                                                                                                                               | Task oriented                                                                                            |
+| Helps beginners or expert users learn a new feature and provide a successful learning experience.                                               | Helps an expert user accomplish a task or troubleshoot an issue.                                         |
+| Follows a carefully managed path, from the start to the end.                                                                                    | Aims for a successful result, and guides the user along the safest, surest way to the goal.              |
+| Eliminates any unexpected scenarios and provides users with a successful finish.                                                                | Alerts the user to the possibility of unexpected scenarios and provides guidance on how to deal with it. |
+| Assumes that users do not have any practical knowledge and must explicitly state any tools, file configurations, conceptual details, and so on. | Assumes that users have the practical knowledge of tools, file configurations, applications, and so on.  |
+
+## Why do I need a how-to?
+
+A how-to guide is often used to help competent users perform a task correctly. It can:
+
+- Demonstrate to your users the different capabilities of your application
+- Guide your users to solve a real-world problem within the application through an ordered series of steps.
+- Help answer specific questions that users may have
+- Make users comfortable using the application.
+- Improve the user experience, and help reduce costs by lowering the number of support requests.
+
+New users who want to solve a problem instantly can also benefit, provided the how-to is well-written and states any prerequisite knowledge required to complete the task.
+
+## Before Writing A how-to
+
+Before you start working on your how-to, identify:
+
+- The main use cases of your application.
+- The different scenarios that your user may encounter in the real world while completing a task. or example, multiple ways through which a task can be completed. If this, then that. In the case of …, an alternative approach is to…
```

Comment:
If this, then that. In the case of …, an alternative approach is to…
Seems it can be a bulleted list of items